packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -21,3 +21,4 @@
 * Implements image streaming.
 * Provides LifecycleOwner implementation for Activities that use the plugin that do not implement it themselves.
 * Implements retrieval of camera information.
+* Initial release of this `camera` implementation.
```

Comment:
I would suggest removing every entry except for this line (since from a client perspective there's nothing that these incremental changes are relative to, this being the first release), and then expanding this a bit to have a summary of what is implemented and a pointer to the readme for details.